pkgs/tools/security/nwaku/default.nix

```diff
@@ -0,0 +1,89 @@
+{
+  stdenv, fetchurl, fetchgit, lib, nim, which, writeScriptBin,
+  # Options: wakunode1, wakunode2, wakubridge
+  makeTargets ? [ "wakunode2" ],
+  # WARNING: CPU optmizations that make binary not portable.
+  nativeBuild ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nwaku";
+  version = "v0.15.0";
+  commit = "e12b7cb4";
+  name = "${pname}-${version}-${commit}";
+
+  src = fetchgit {
```

Comment:
Please use fetchFromGitHub

Reply:
Sure.

Reply:
BTW, what is the benefit of `fetchFromGitHub` over `fetchgit`?